File: erezervisi_desktop/erezervisi_desktop/lib/models/responses/guest/guest_get_dto.dart
class GuestGetDto {
  late num id;
  late String fullName;
  late String phone;
  late String email;
  String? accommodationUnitTitle;
  String? review;

  GuestGetDto(
      {required this.id,
      required this.fullName,
      required this.phone,
      required this.email,
      this.accommodationUnitTitle,
      this.review});

  factory GuestGetDto.fromJson(Map<String, dynamic> json) {
    return GuestGetDto(
      id: json['id'] as num,
      fullName: json['fullName'],
      phone: json['phone'],
      email: json['email'],
      accommodationUnitTitle: json['accommodationUnitTitle'],
    );
  }
}

File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/Guest/GuestGetDto.cs
namespace eRezervisi.Common.Dtos.Guest
{
    public class GuestGetDto
    {
        public long Id { get; set; }
        public string FullName { get; set; } = null!;
        public string Phone { get; set; } = null!;
        public string Email { get; set; } = null!;
    }
}
